test-result-summary-client/src/Build/ReleaseSummary.jsx

```diff
@@ -50,17 +52,26 @@ export default class ReleaseSummary extends Component {
                         javaVersion,
                         tests = [],
                     }) => {
-                        const buildInfo = `\n[**${buildName}**](${buildUrl})`;
-                        const buildResultStr =
+                        const buildInfo = `${nl}[**${buildName}**](${buildUrl})`;
+                        let buildResultStr =
                             buildResult === 'UNSTABLE'
-                                ? ` ⚠️ ${buildResult} ⚠️\n`
-                                : ` ❌ ${buildResult} ❌\n`;
+                                ? ` ⚠️ ${buildResult} ⚠️${nl}`
+                                : ` ❌ ${buildResult} ❌${nl}`;
                         if (buildName.startsWith('Test_openjdk')) {
+                            const buildData = await fetchData(
+                                `/api/getData?_id=${_id}`
+                            );
```

Comment:
Do we need another API call? Does `failedBuilds ` contain the information?

Reply:
Oh yes, I must have missed that! Although, some children do not have `rerunLink`. That' fine, right?
<img width="395" alt="image" src="https://user-images.githubusercontent.com/37882255/161108927-eda8c966-2ac7-48e7-8fbe-b9c53c0dbd0f.png">

Reply:
> some children do not have rerunLink. That' fine, right?

Yes, that's normal. Some test build may fail before the rerun link is generated.